### DISCUSSION TITLE
Samsung Galaxy S22 flickering (OpenGL)

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#546] [Q&A] [answered]
There was a report with video about map background flickering on Samsung Galaxy S22 Ultra:

- https://github.com/mapsforge/vtm/discussions/1128#discussioncomment-9532710

Does anyone have this phone or seen anything similar?

**Comment by MotoUKRider** [comment]
@devemux86 Galaxy S20 on Android 13 here - no such issue with UK, Netherlands or German maps (included the specific area the user posted).

What I *do* see, and this also occurs on the Desktop Cruiser (so could be a rendering "feature") are tiny hair-lines that (dis)appear as I move the map - they tend to occur just over a label (land or water).

**Reply by devemux86** [reply]
@MotoUKRider Can you post some screenshots?

**Reply by MotoUKRider** [reply]
@devemux86 Yes, I suppose that would be helpful!

It only seems to happen in my MotoRider theme (not Biker).  But I can't immediately see why it is happening.
![cruiser-line-render](https://github.com/devemux86/cruiser/assets/125696109/68851056-97ff-4151-8b39-1858da4556a4)

They (dis)appear as I move my finger across the map, much the same as in Desktop.  I don't think there are conflicting objects as at high-zoom there's nothing else there. Perhaps there's a conflicting rule that I've created although the theme is mostly just a colour-swap from "Biker"... Mostly.

**Reply by devemux86** [reply]
This seems like an OpenGL issue.

Latest desktop does not have OpenGL (it is hidden), what version do you use?

**Comment by MotoUKRider** [comment]
Currently on 4.0.1b4 Desktop.  It's happened for quite a few versions, I always assumed that it was an undocumented feature!

**Comment by DolomitesTom** [comment]
No flickering. Only last weekend I had the phenomenon that I had already reported in the past, that the cursor (GPS position) jumps back and forth, which meant that I missed the right exit. 
But it was only a temporary problem.

**Reply by devemux86** [reply]
@DolomitesTom thanks, do you have Galaxy S22?

**Reply by DolomitesTom** [reply]
No, also 2 older Samsung devices.

**Comment by devemux86** [comment, marked as answer]
Let's close it here, as Cruiser 5 will have a new map engine without all these strange problems.
